src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_img2img.py

```diff
@@ -207,7 +186,12 @@ def __init__(
             feature_extractor=feature_extractor,
         )
         self.vae_scale_factor = 2 ** (len(self.vae.config.block_out_channels) - 1)
-        self.register_to_config(requires_safety_checker=requires_safety_checker)
+
+        vae_feature_extractor = VaeImageProcessor(
+            vae_scale_factor =self.vae_scale_factor)
+        self.register_to_config(
+            requires_safety_checker=requires_safety_checker,
+            vae_feature_extractor = vae_feature_extractor)
```

Comment:
```suggestion
            )
```

Reply:
Let's not save the feature extractor as it's an object and not a config (it's not JSON serializable)